pydantic/types.py

```diff
@@ -687,6 +689,110 @@ def get_secret_value(self) -> bytes:
         return self._secret_value
 
 
+class PaymentCardBrand(Enum):
+    amex = 'American Express'
+    mastercard = 'Mastercard'
+    visa = 'Visa'
+    other = 'other'
+
+    def __str__(self) -> str:
+        return self.value
+
+
+class PaymentCardNumber(str):
+    """
+    Based on: https://en.wikipedia.org/wiki/Payment_card_number
+    """
+
+    strip_whitespace = True
+    min_length: int = 12
```

Comment:
Might be better to annotate these as `typing.ClassVar`s?

Reply:
works for me: aab24703c78e8d853c63376cafbd71f466894a67